### PR TITLE
[META] PR for discussion

### PR DESCRIPTION
**Pull request by matmair**
See #37 
THIS WILL NOT BE MERGED

### CODE REVIEW

File: InvenTree/web/src/src/contex/ApiState.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import create from 'zustand';
```
Comment (wolflu05) [inline InvenTree/web/src/src/contex/ApiState.tsx]: could it be, that there is a spelling mistake in the name of the folder this file is in?

Comment (matmair) [reply InvenTree/web/src/src/contex/ApiState.tsx]: @wolflu05 `zustand` is the state managment libary - [see](https://www.npmjs.com/package/zustand)

Comment (wolflu05) [reply InvenTree/web/src/src/contex/ApiState.tsx]: I know, I'm beginning to learn Zustand. But from my current understanding, contex (without t) makes no sense for me.

Comment (matmair) [reply InvenTree/web/src/src/contex/ApiState.tsx]: @wolflu05 I do not really understand what you mean by this sentence. Should we switch to German?
`create` is used to create the store for the states.

Comment (wolflu05) [reply InvenTree/web/src/src/contex/ApiState.tsx]: Sorry, this question has nothing to do with the line where I commented on. Github doesn't allow to comment on folder names. I mean if this is a spelling mistake in the name of the folder this file (where I commented on) is in. You called it contex (without an t at the end) contex~~t~~, isn't the t missing here?

Comment (matmair) [reply InvenTree/web/src/src/contex/ApiState.tsx]: @wolflu05  thanks, I changed it.
Commenting on PRs is sometimes wiered on GitHub

**Comment by wolflu05** [comment]
Do I got this right, that you are currently switching out maintineui with bootstrap? Are there any reasons, just curious?

**Comment by matmair** [comment]
@wolflu05 I am testing out compatibility with plugins that assume bootstrap is available.

**Comment by wolflu05** [comment]
@matmair And what about using [dangerouslysetinnerhtml](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) for legacy plugins, and current plugins can just provide an bundle file which has a default export which is included via runtime import() and rendered as component?

_Originally posted by @wolflu05 in https://github.com/inventree/InvenTree/pull/4114#issuecomment-1368508498_

**Comment by matmair** [comment]
@wolflu05 I think for a start we will use iframes and then work around arising problems.

**Comment by wolflu05** [comment]
So v1 compatible plugins can't use react directly, they need to provide their own build or use something like preact what I already do?

**Comment by matmair** [comment]
@wolflu05 that part of the infrastructure is not done yet so I don't really know. We will release the frontends in parallel so we can check for that stuff.

**Pull request closed without merging**